### PR TITLE
Fix epoll and socket resource leak issue.

### PR DESCRIPTION
**Pull request by liuh-80**
#### Why I did it
Fix epoll and socket resurce leak issue:
[chassis] Too many open files error and unable to connect to redis socket error
https://github.com/Azure/sonic-buildimage/issues/10870

The reason of this issue is in SWIG any method return a new object need decorate with %newobject, so SWIG will generate code to release C++ object when python wrapper object released:
https://www.swig.org/Doc4.0/SWIGDocumentation.html#Customization_ownership

#### How I did it
Update swsscommon.i to decorate return new object methods with %newobject

#### How to verify it
Pass all test case.

Run following code in python and validate there is no epoll and socket leak:

from swsscommon.swsscommon import SonicDBConfig
from swsscommon.swsscommon import SonicV2Connector
import gc

SonicDBConfig.load_sonic_global_db_config()
SonicDBConfig.get_ns_list()
db = SonicV2Connector(use_unix_socket_path=True, namespace='')
db.connect("CONFIG_DB")
db.get_redis_client("CONFIG_DB")
client = db.get_redis_client("CONFIG_DB")
client.pubsub()
client.pubsub()
client.pubsub()
client.newConnector(0)
client.newConnector(0)
client.newConnector(0)

gc.collect()

#### Which release branch to backport (provide reason below if selected)

- [ ] 201811
- [ ] 201911
- [ ] 202006
- [ ] 202012
- [ ] 202106
- [x] 202111

#### Description for the changelog
Fix epoll and socket resurce leak issue:
[chassis] Too many open files error and unable to connect to redis socket error
https://github.com/Azure/sonic-buildimage/issues/10870

### CODE REVIEW

File: pyext/swsscommon.i [hunk 1]
```diff
@@ -158,6 +158,14 @@ T castSelectableObj(swss::Selectable *temp)
     %template(hgetall) hgetall<std::map<std::string, std::string>>;
 }
 
+%extend swss::PubSub {
```
Comment (liuh-80) [inline pyext/swsscommon.i:161]: WIll delete this code.
Also will check if other method need decorate with %newobject

Comment (liuh-80) [reply pyext/swsscommon.i]: Fixed, these 2 methods are all methods return new object in sonic-swss-common.

**Comment by liuh-80** [comment]
sairedis build failed because a known issue. will retry till it success.

**Comment by abdosi** [comment]
can we add UT that can conver this scenario.

**Review by kcudnik — APPROVED** [review]

**Review by prgeor — APPROVED** [review]

**Review by qiluo-msft — APPROVED** [review]

**Comment by qiluo-msft** [comment]
I see the manual testcase in "How to verify it" section. Is it possible to automate the test as unit test or sonic-mgmt test?

**Pull request merged**